src/main/webapp/WEB-INF/index.html

```diff
@@ -12,8 +12,19 @@
 <div class="container">
   <div class="tasks-container">
 
+    <div class="banner" id="info">
+      <p class="banner-text"> Hey! Select some tasks you would like to move to your calendar, and hit schedule! You can also select the timerange of scheduling, and set duration for tasks.</p>
```

Comment:
Would make this a bit more concise and e.g. remove the "Hey". Maybe you can get some inspiration from Google's public UX writing guidelines.
https://material.io/design/communication/writing.html#principles